translations/pt-br/04-ferramentas-utilizadas.md

```diff
@@ -0,0 +1,142 @@
+- **Forking**: O GitHub permite que os utilizadores criem uma cópia de um repositório, conhecida como fork. Isso permite que eles façam alterações na base de código sem afetar o projeto original. Uma vez que as alterações estejam concluídas, eles podem enviar um pedido de pull para que as suas alterações sejam revisadas e potencialmente mescladas na base de código principal.
+- **GitHub Actions**: O GitHub Actions é uma poderosa ferramenta de automação que permite criar fluxos de trabalho personalizados para automatizar tarefas como a construção, teste e implantação do seu código.
+
+### Começando com Git e GitHub
+
+Para começar com Git e GitHub, siga estes passos:
+
+1. **Instalar o Git**: Baixe e instale o Git no seu computador visitando o site oficial do Git (https://git-scm.com/) e seguindo as instruções de instalação para o seu sistema operacional.
+2. **Configurar o Git**: Após instalar o Git, abra um terminal ou prompt de comando e configure as informações do seu usuário Git executando os seguintes comandos:
+
+```bash
+git config --global user.name "Seu nome"
+git config --global user.email "seu.email@exemplo.com"
+```
+
+Substitua "Seu nome" pelo seu nome completo e "seu.email@exemplo.com" pelo endereço de e-mail que você usou para se inscrever no GitHub.
+
+#### Praticando Contribuições com o VS Code
+
+Para praticar o uso do GitHub, vamos passar por criar um repositório, clonar o repositório, fazer alterações e empurrar essas alterações.
+
+1. **Crie um repositório**: Faça login na sua conta do GitHub e navegue até [https://github.com/new](https://github.com/new) ou clique no ícone "+" no canto superior direito e selecione "Novo repositório" no menu suspenso.
+
+Escollha você mesmo como o proprietário do repositório, e digite um nome para o seu repositório. Para este exemplo, usaremos `practice-repository`
+
+![new repo screenshot](../../images/new-repo.png)
+
+Escollha se você quer que seja público ou privado. Você também pode escolher inicializar o repositório com um arquivo README, um arquivo .gitignore e/ou uma licença.
+
+Por exemplo, inicialize o repositório com um arquivo README. 
+
+2. **Clone o repositório**: Uma vez que você tenha criado um repositório, você pode cloná-lo para a sua máquina local executando o seguinte comando no seu terminal ou prompt de comando:
+
+```bash
+git clone https://github.com/username/repository-name.git
+```
+
+Substitua "username" pelo seu nome de utilizador do GitHub e "repository-name" por `practice-repository`.
+
+3. **Acesse o seu repositório no seu editor de código**: Para este exemplo, estamos usando o VS Code. Abra o VS Code, navegue até a guia de arquivos e abra o seu `practice-repository`.
+
+4. **Faça alterações e confirme**: Deve haver um arquivo no seu repositório: `README.md`. Dentro desse arquivo, abaixo de `# practice-repository`, digite 'hello world!' Após ter feito alterações, você pode prepará-las e confirmá-las.
```

Comment:
```md
Após ter feito alterações, você pode prepará-las e confirmá-las.
```
"stage and commit them"
Well I guess "commit" should be revised around to make sure it's always utilized the same word, and not a contextual translation. We should commit to a unique word that means commit in the git sense, or just plainly use "commit". Here is translated as "confirmar" and a few lines above is "aplicar" / "comprometer"... What do you think?